cmd/discovery/cmd/send.go

```diff
@@ -7,17 +7,17 @@ package cmd
 
 import (
 	"fmt"
-
-	"github.com/seeleteam/go-seele/p2p/discovery"
+	"net"
 
 	"github.com/spf13/cobra"
+
+	"github.com/seeleteam/go-seele/common"
+	"github.com/seeleteam/go-seele/p2p/discovery"
 )
 
-var
-(
-	myport *string
-	targetid *string
-	targetport *string
+var (
+	addr          *string
+	bootstrapNode *string
```

Comment:
nice to add comments for this var and what the difference between addr and bootstrapNode